libs/api/src/lib/context.ts

```diff
@@ -115,6 +116,7 @@ export interface DataLoaderContext {
   readonly eventById: DataLoader<string, Event | null>
 
   readonly commentsById: DataLoader<string, Comment | null>
+  readonly commentRatingSystemAnswers: DataLoader<1, CommentRatingSystemAnswer[]>
```

Comment:
it needs a key for the dataloader as a cache, but it doesn't have one. Forced it to. be always 1.

With API v2 e can use the caching service instead.